src/@types/index.d.ts

```diff
@@ -0,0 +1,75 @@
+export {}
+
```

Comment:
Don't use inline imports:
```suggestion

import SinglePoll from '../models/SinglePoll'
import { TOCCode } from '../data/TocData'
```

Reply:
Oops, was required in an older version of the file. I'll change them over.